src/main/java/net/sf/jabref/logic/exporter/BibDatabaseWriter.java

```diff
@@ -158,6 +158,8 @@ public E savePartOfDatabase(BibDatabaseContext bibDatabaseContext,
 
         session = saveSessionFactory.createSaveSession(preferences.getEncodingOrDefault(), preferences.getMakeBackup());
 
+        writeDatabaseID(bibDatabaseContext.getDatabase().getDatabaseID());
```

Comment:
I think plain bibtex is used when the user want's to export the bib file and send it to somebody not using JabRef (e.g. when submitting to a journal). Then all the JabRef things like metadata are removed.